spec/lib/movement_api/shapefile_importers/congressional_district_importer_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+require "movement_api/shapefile_importers/congressional_district_importer"
+
+module MovementApi
+  module ShapefileImporters
+    describe CongressionalDistrictImporter do
+      let(:file) { "spec/support/fixtures/shapefiles/congressional_districts/file.shp" }
+
+      subject { CongressionalDistrictImporter.new(file) }
+
+      it "has the right number of districts in the shapefile" do
+        RGeo::Shapefile::Reader.open('spec/support/fixtures/shapefiles/congressional_districts/file.shp', srid: 4326) do |file|
+          expect(file.num_records).to eq 4
+        end
+      end
+
+      describe "#call" do
+        it "creates CongressionalDistrict models" do
+          expect { subject.call }.to change{CongressionalDistrict.count}.from(0).to(3)
+        end
+      end
```

Comment:
I'm just thinking something like

```Ruby

mock_shapefile = []

x.times do
  mock_attributes = {
    "STATEFP" => "50"
    "CD114FP" => "ZZ"
    "CDSESSN" => "something"
    ...
  } 
  mock_record = double("Something", attributes: mock_attributes)
  mock_shapefile.push(mock_record)
end

allow(RGeo::Shapefile::Reader).to receive(:open).and_return(mock_shapefile )
```

It's not as simple as I'd like it to be, but might be simpler than creating files for each combination.